modules/proxy-certificates.adoc

```diff
@@ -0,0 +1,75 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="proxy-certificates_{context}"]
+= Proxy certificates
```

Comment:
@danehans Can you please review what's in place for proxy certs so far?

Reply:
@danehans Can you please provide feedback by end of week?

Reply:
@ahardin-rh I apologize for not getting you feedback earlier. The proxy-related docs look good. The process outlined below properly describes how to manage proxy certs during *runtime*. Note that `additionalTrustBundle` of the installer config is used to specify any proxy trusted ca certs during install time. For example:
```
$ cat install-config.yaml 
<SNIP>
proxy:
  httpProxy: http://<HTTP_PROXY>
  httpsProxy: https://<HTTPS_PROXY>
additionalTrustBundle: |
    -----BEGIN CERTIFICATE-----
   <MY_HTTPS_PROXY_TRUSTED_CA_CERT>
    -----END CERTIFICATE-----
<SNIP>
```